api/MockApi/ApiInstance.cs

```diff
@@ -0,0 +1,73 @@
+﻿using SynthesisAPI.Modules;
+using SynthesisAPI.Runtime;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using SynthesisAPI.EnvironmentManager;
+using UnityEngine.UIElements;
+
+#nullable enable
+
+namespace MockApi
+{
+    public class ApiInstance : IApiProvider
```

Comment:
This class should be deleted. We already have a mock API provider in `MockApi.cs`